src/scales/scale.category.js

```diff
@@ -47,6 +47,13 @@ module.exports = Scale.extend({
 		me.ticks = (me.minIndex === 0 && me.maxIndex === labels.length - 1) ? labels : labels.slice(me.minIndex, me.maxIndex + 1);
 	},
 
+	convertTicksToLabels: function() {
```

Comment:
`convertTicksToLabels` seems like maybe a funny place to do this? I would probably have chosen `initialize` or `update`